src/scripts/main.js

```diff
@@ -1,3 +1,46 @@
 'use strict';
 
-// write your code here
+const promise1 = new Promise((resolve) => {
+  const logo = document.querySelector('.logo');
+
+  logo.addEventListener('click', () => {
+    resolve('Promise was resolved!');
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(new Error('Promise was rejected!'));
+  }, 3000);
+});
+
+const createSuccessMessage = (data) => {
+  const body = document.querySelector('body');
+  const successDiv = document.createElement('div');
+
+  successDiv.classList.add('message');
+  successDiv.innerText = data;
+  body.appendChild(successDiv);
+};
+
+const createErrorMessage = (err) => {
+  const body = document.querySelector('body');
+  const errorDiv = document.createElement('div');
+
+  errorDiv.classList.add('error-message');
+  errorDiv.innerText = err;
+  body.appendChild(errorDiv);
+};
```

Comment:
i'd DRY it even more :)
```suggestion
const createSuccessMessage = (
  type: 'data' | 'error', data
) => {
  const body = document.querySelector('body');
  const div = document.createElement('div');
  const classname = type === 'message'
    ? 'message'
    : 'error-message';

  div.classList.add(classname);
  div.innerText = data;
  body.appendChild(div);
};
```